gtk4/src/subclass/layout_manager.rs

```diff
@@ -29,7 +29,10 @@ pub trait LayoutManagerImpl: LayoutManagerImplExt + ObjectImpl {
         self.parent_create_layout_child(layout_manager, widget, for_child)
     }
 
-    fn layout_child_type() -> glib::Type;
+    /// Only set if the child implemented LayoutChildImpl
+    fn layout_child_type() -> Option<glib::Type> {
+        None
```

Comment:
Should still call the parent one :)

Reply:
There's no parent one it's not a method but it sets the type on the Class struct with
```
        if let Some(type_) = T::layout_child_type() {
            klass.layout_child_type = type_.to_glib();
        }
```
Setting a `layout_child_type` to a widget instead of LayoutChild requires having `layout-child` property which is the widget; not something the user often has to do that's why it's None by default